packages/privacy-toolset/src/cookie-banner/styles.scss

```diff
@@ -0,0 +1,256 @@
+// Granular consent banner (v2).
+$granular-banner-max-width: 1024px;
+
+// Colors.
+$cookie-banner-bg-color: #FFFFFF;
+$cookie-banner-text-color: #000000;
+$cookie-banner-text-muted-color: #444444;
+$cookie-banner-link-color: #117ac9;
```

Comment:
Extreme Nitpick Alert: probably should stick to either all capital letters or all lowercase when writing hex codes 😄

Reply:
I'll take care of this when adjusting the code to scss ruleset requirements in the repo, thanks!

Reply:
It's actually enforced by stylelint, all changed to lowercase